shell/client/admin-client.js

```diff
@@ -1182,11 +1182,24 @@ const adminRoute = RouteController.extend({
                   let looksGood;
                   if (error) {
                     looksGood = false;
+                    console.log("Sandstorm WILDCARD_HOST self-test failed. Details:");
+                    console.log(error);
+                    console.log(
+                      "Further details may be available in the JS console provided by your web " +
```

Comment:
FWIW what I'm trying to say is:

- The JS console will have text that comes from Sandstorm.

- The JS console MIGHT have other, colorized text that comes from your web
browser.

- Sandstorm can't see that error, but you, a human operator of the web
browser, can indeed see that error!

- So you should look at it and see what it says.

See also the screenshot I provided, where Chromium provides more details
about the error than Sandstorm can perceive.

I'm 100% open to better ways to phrase this; hopefully the general intent
is now clear.